### PR TITLE
Milestone 1:New York Times Homepage

### PR DESCRIPTION
**Pull request by Murodjon000**
Hi TSE, 👋

I used a flexbox and grid to make this project. And I also made it fully responsive. I think this project helps me to improve my knowledge about Floating, Flexbox, and Grid

Best,

Murodjon 🙂

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## :stop_sign: Invalid

### Hi @Murodjon000 :wave:  

I am marking this submission as invalid, but not to worry. 

Read through and implement the instructions on this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/), you will be fine :100: 

### However as a pointer:

 - [x] There is no custom README, in this repository.

- [x] Your pull request should have a meaningful title and short summary. use this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) as a guide.

#### That is it, implement the changes and submit another form for a code review.

#### ``` Have a great day ``` :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

**Comment by Murodjon000** [comment]
### **New York Times Homepage**

**In this Project**:

-       -We build it with Flexbox
-       -We build it with Grid
-       -We build it with a fully responsive design

File: index.html
Comment (jubaan) [inline index.html]: - [x] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [x] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [x] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [x] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [x] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [ ] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [ ] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [ ] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [ ] This should be links `<a>`

File: index.html
Comment (jubaan) [inline index.html]: - [x] The titles and the images should be lins `<a>`

File: style.css
Comment (jubaan) [inline style.css]: - [x] This rule is repeated numerous time, create a helper CSS class for it and apply it to the elements that use it.

File: style.css
Comment (jubaan) [inline style.css]: - [x] This rule is repeated numerous time, create a helper CSS class for it and apply it to the elements that use it.

File: style.css
Comment (jubaan) [inline style.css]: - [x] This rule is repeated numerous time, create a helper CSS class for it and apply it to the elements that use it.

File: style.css
Comment (jubaan) [inline style.css]: - [x] This could be simplified, look for patterns similar to this, and change or create a helper CSS Class
```css
BAD
 .editor__pick {
    display: none;
  }

  .advert__box {
    display: none;
  }

  .most__popular {
    display: none;
  }

GOOD

 .editor__pick,
 .advert__box,
  .most__popular {
    display: none;
  }
```

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @Murodjon000  . Thanks for submitting you project!

Great thing this project has and has done well:
:100: is great the project uses media queries even if it was not a requirement
:100: good use of BEM CSS naming convention

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] README has no description of the project.

- [ ] The Live Demo link is not working

- [ ] The Link to the issues page is broken

- [ ] The link to author github account takes to the repository instead of the profile

- [ ] NYT Logo is not centered, as well as the content column
image
image

- [ ] The aside editors pick element is not in the correct position and is invading the content column space
image

- [ ] The project is missing an embedded video, and images that are along the content column, as well as the content text of each missing section.

- [ ] Missing element

image

- [ ] Missing element

image

- [ ] Missing element

image

- [x] Missing element

image

- [x] Missing element

image

- [x] Missing element

image

- [x] Missing element

image

- [x] Missing fixed footer
image

- [x] Use paths that start with dot, like `./` for files stored in the repository like stylesheets
```
// Bad Paths
<link rel="stylesheet" href="/assets/stylesheets/main.css">
<link rel="stylesheet" href="/assets/js/bootstrap.min.js">
<img src="/imgs/myimage.png">


// Good Paths
<link rel="stylesheet" href="./assets/stylesheets/main.css">
<link rel="stylesheet" href="./assets/js/bootstrap.min.js">
<img src="./imgs/myimage.png">
```

- [x] Use semantic HTML for `<header>`, and `<main>`

- [x] Keep your code DRY in the CSS file by separating repeated rules and creating helper CSS classes. See inline comments


### [OPTIONAL]  :yellow_circle:  

- [x] Dont leave the `alt` attribute black, it should be a short but meaningful description.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @Murodjon000**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] the logo isn't centered it's now leaning to right a little bit.

- [ ] you mustn't use a screenshot in the theory of inflation section.

- [ ] in the More in Space and Astronomy section things are not aligned
fkjy
 

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Comment by mimipeshy** [comment]
### STATUS :CHANGES REQUIRED :red_circle:  :recycle: 
Hey team :wave: :wave:, a very good job so far :muscle: , you are almost close to getting approved for this project, but lets fix a few more things. 
- [ ] The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))
- Add a `<a href>` tag to the image link to achieve this.

- [ ] You are missing the following section on your page :arrow_down: 
- In the `More in Space and astronomy section` add one more row with three column item
- Add the `Editors section` below it

www nytimes com_2014_03_18_science_space_detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang html (2) (1)


That's it :smile: , kindly submit another review when done. :pray: 
Regards
Peris Ndanu
Feel free to slack me anytime if you have a question

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey team, 👋

You have done an amazing job implementing the project requirements for this project.

However, there are still some changes 👇 you need to implement to approve this request 🔍

### Changes Required ♻️ 

- [ ] As already stated by the second reviewer, you are missing the section titled **Theory of Inflation**. See image below. You use `css grid` to implement this feature with ease.

screenshot-www nytimes com-2020 11 26-12_07_40


#### Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime**
I would be happy to help! 👩‍💻 

Happy coding 🤓

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on implementing all project requirements and getting this project approved. :+1:

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
